src/js/components/languageToggleBtn.js

```diff
@@ -1,6 +1,14 @@
-import { toggleLanguage, currentLanguage } from '../helpers/languageManager';
+export let currentLanguage = 'sv';
+
+export const toggleLanguage = (button) => {
+  currentLanguage = currentLanguage === 'sv' ? 'en' : 'sv';
+  button.textContent = currentLanguage === 'sv' ? 'In English' : 'Till Svenska';
+};
+
 export const createLanguageToggleBtn = (initialLanguage, onToggle) => {
-  const button = document.createElement('button');
+  const buttonContainer = document.createElement( 'div' )
+  
+  const button = document.createElement( 'button' );
```

Comment:
If buttonContainer is supposed to contain the button variable... maybe: buttonContainer.createElement( 'button' ); would fix this issue. :) Otherwise really good job here Elena!

Reply:
I agree with Taru, very impressive work and the page works as expected, according to 'Definition of done'.
I think it is ready to be merged.

Reply:
thank you for your review! I have removed the button container from the code. :)